Landscape layout 2

Lucrezia-Cester opened the pull request.

Lucrezia-Cester committed:
landscape.html

Lucrezia-Cester committed:
delete landscape.hmtl

Lucrezia-Cester committed:
add landscape.html

Lucrezia-Cester committed:
added landscape.yml

Lucrezia-Cester committed:
created landscape_l.html

Lucrezia-Cester committed:
landscape folder

Lucrezia-Cester committed:
added landscape.md

Lucrezia-Cester commented:
Hi @tomaroberts, this is now ready for review. I have created a new PR since I changed many things compared to the other landscape PR I had opened.

tomaroberts commented on _includes/landscape.html:
```diff
@@ -0,0 +1,17 @@
+	<div class="col-sm-4 pb-5">
+	  <div class="platform">
+		<a href="{{ platform.platform }}">
+		  <img style="width: 300px; height:200px; padding = 1px; border: 8px solid transparent; background: linear-gradient(90deg, #e1f0d0 100%, #ce00 100%)" src="{{ platform.image }}" alt="">
```
The aspect ratio of the images is non-square, which looks bad. This is because you have both width and height tags hard-coded. If you remove the height tag, then the code should retain the aspect ratio of the underlying images. You will need to standardise the aspect ratio of the images you use, otherwise the images will be different sizes on the page. The easiest way to do this might be to just ensure all the images are say 400px x 400px, or whatever looks best.

I'm also not keen on the green border around the images, but I think the style of the site has yet to emerge, so for now I would probably just use a dull thin black line or something until we figure out what styling we want to use.

tomaroberts commented on _includes/landscape.html:
```diff
@@ -0,0 +1,17 @@
+<!-- Landscape -->
+<h5 class="text-muted">Click on the platforms to find out more about them.</h5>
```
You could replace this line with the `text:` entry in your .yml file if you want.

tomaroberts commented on thelandscape.md:
```diff
@@ -0,0 +1,5 @@
+---
+layout: landscape_l
```
Can you rename this "landscape" and the update the accompanying places it's used in the code, just to maintain code convention.

tomaroberts commented on _data/landscape.yml:
```diff
@@ -0,0 +1,41 @@
+en: &DEFAULT_EN
+
+  landscape:
+    text: "Click on the platform to find out more about each paltform."
```
Typo: platform

tomaroberts reviewed (changes requested):
Hi Lu, thanks for this. I have a meeting so slightly in a rush, but...

- Broad issue: we need to decide as a group exactly what we want in this page. But the platforms you've listed are a logical starting point. I wonder if we want to sub-divide into Internal and External platforms.
- The images need to be hyperlinks to pages specifically about the platforms. I think we should keep people on the site rather than just linking to the external pages, so we could have little summaries of the platforms and how we work with them/use them.
- If I shrink my browser window, the images overlap, which they shouldn't. This is a bootstrap issue. I have the same problem on the About Us page. It's unlikely people will see this issue, so not super urgent, but something we should fix.

If I think of anything else, I'll add some more comments. Thanks for this - already raised various questions in my mind!

Lucrezia-Cester committed:
removed height tag

Lucrezia-Cester committed:
remaned landscpae_l as landscpae

Lucrezia-Cester committed:
renamed landscpae_l as landscape

Lucrezia-Cester committed:
changed image height dimensions

Lucrezia-Cester committed:
updated figures names

Lucrezia-Cester committed:
updated pictures with different px sizes

Lucrezia-Cester committed:
deleted old size aide pic

Lucrezia-Cester committed:
deleted old size xnat pic

Lucrezia-Cester committed:
delete old cockstack file

Lucrezia-Cester committed:
deleted old mlops file

Lucrezia-Cester committed:
added link from aide image in platforms to new page 

now if you click on the aide platform it brings you a new page where there is more information about the platform

Lucrezia-Cester committed:
deleted pic

tomaroberts commented:
Hi Lu – there is a problem with your latest set of commits. When I run the site based on this branch, it looks like this:

<img width="1106" alt="image" src="https://user-images.githubusercontent.com/15610752/158205810-13e82b5c-e015-402a-b78d-332caffce08f.png">

I think you have deleted the images from the repo?

Some other requests. Can you:
- [ ] update the navbar code so that the PLATFORMS button redirects to `landscape.html`
- [ ] based on the latest Wednesday workshop, add the platform names to the subnav bar. So, it should be like:

| ABOUT US | PEOPLE | PLATFORMS | POLICY | OUR WORK | BLOG |
| - | - | - | - | - | - |
| LANDSCAPE | AIDE | FLIP | CogStack | XNAT | MLOps |

- [ ] above the images, add some introductory text giving a general sense of what the page is about and we are making these platforms.

Lucrezia-Cester committed:
this ensures the landscape is the landing page of platforms

Lucrezia-Cester committed:
deleted wrong pic directory

Lucrezia-Cester committed:
added cogstack image

Lucrezia-Cester committed:
added aide

Lucrezia-Cester committed:
mlops

Lucrezia-Cester committed:
xnat

Lucrezia-Cester commented:
Hi @tomaroberts, I have addressed your comments. It is now ready for review.

Lucrezia-Cester committed:
updated subnav platform

This now ensures there are all the platform in the landscape subbar. Also, when clicking on AIDE platform, it redirects you to the AIDE.html page

Lucrezia-Cester committed:
changed name of landascape layout

I had given the same name to the layout and to the includes so it was rendering weird

Lucrezia-Cester committed:
added AIDE platform

It didn't make sense to separate the AIDE_landscape platform PR from the landscape PR because there was too much code they shared. So I am adding the code of the other PR into this one

Lucrezia-Cester committed:
updated subnavbar
